app/views/repositories/revision.html.erb

```diff
@@ -27,28 +27,29 @@ See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
 
-<% content_for :action_menu_specific do %>
-  &#171;
-  <% unless @changeset.previous.nil? -%>
-    <%= link_to_revision(@changeset.previous, @project, :text => l(:label_previous)) %>
-  <% else -%>
-    <%= l(:label_previous) %>
-  <% end -%>
-  |
-  <% unless @changeset.next.nil? -%>
-    <%= link_to_revision(@changeset.next, @project, :text => l(:label_next)) %>
-  <% else -%>
-    <%= l(:label_next) %>
-  <% end -%>
-  &#187;&nbsp;
-
-  <%= form_tag({:controller => '/repositories', :action => 'revision', :project_id => @project}, :method => :get) do %>
-    <%= text_field_tag 'rev', @rev, :size => 8 %>
-    <%= submit_tag 'OK', :name => nil, class: 'button -highlight' %>
-  <% end %>
+<% content_for :toolbar do %>
+  <li class="toolbar-item">
+    <% if @changeset.previous.nil? %>
+      <button class="button" disabled="disabled"><%= l(:label_previous) %></button>
+    <% else %>
+      <%= link_to_revision(@changeset.previous, @project, text: l(:label_previous), class: 'button') %>
```

Comment:
Well, You cannot disabled a `a`, you can of course set an attribute `disabled` and then add the CSS to make it seem like it is disabled, but from as spec perspective, HTML does not allow `disabled` on `a`.

That's why I used a button instead. What on could potentially do was to use `button_to` in both cases.